src/Workspaces/Core/Portable/Shared/Extensions/SyntaxGeneratorExtensions_CreateEqualsMethod.cs

```diff
@@ -255,8 +254,7 @@ public static IMethodSymbol CreateEqualsMethod(this Compilation compilation, Imm
             SyntaxGenerator factory,
             Compilation compilation,
             INamedTypeSymbol containingType,
-            ImmutableArray<ISymbol> members,
-            CancellationToken cancellationToken)
```

Comment:
:memo: This change unnecessarily complicated this review

Reply:
broken windows.